### PR TITLE
Add Sysinternals Sysmon to emerging applications

### PR DESCRIPTION
**Pull request by MarioHewardt**
Hi, All - I would like to add [Sysmon for Linux](https://github.com/Sysinternals/SysmonForLinux) to the list of emerging applications.

### CODE REVIEW

File: src/pages/applications.js [hunk 1]
```diff
@@ -30,7 +30,7 @@ import majorProjects from "../data/major-projects";
 import emergingProjects from "../data/emerging-projects";
```
Comment (vannyle) [inline src/pages/applications.js:30]:
```suggestion
import sysinternalsLogo from "../assets/projects-logos/sysinternals.svg";
import emergingProjects from "../data/emerging-projects";
```

File: src/pages/applications.js [hunk 2]
```diff
@@ -30,7 +30,7 @@ import majorProjects from "../data/major-projects";
...
 
 const ProjectDescriptions = () => {
-  const { tetragonLogo, loxiLBLogo, pwruLogo, wachyLogo, skywalkingLogo, pulsarLogo, deepflowLogo, inspektorGadgetLogo, bpftraceLogo, eunomiaLogo, kindlingLogo } = useStaticQuery(graphql`
+  const { tetragonLogo, loxiLBLogo, pwruLogo, wachyLogo, skywalkingLogo, pulsarLogo, deepflowLogo, inspektorGadgetLogo, bpftraceLogo, eunomiaLogo, kindlingLogo, sysinternalsLogo } = useStaticQuery(graphql`
```
Comment (vannyle) [inline src/pages/applications.js:33]:
```suggestion
  const { tetragonLogo, loxiLBLogo, pwruLogo, wachyLogo, skywalkingLogo, pulsarLogo, deepflowLogo, inspektorGadgetLogo, bpftraceLogo, eunomiaLogo, kindlingLogo } = useStaticQuery(graphql`
```

File: src/pages/applications.js [hunk 3]
```diff
@@ -87,6 +87,11 @@ const ProjectDescriptions = () => {
         gatsbyImageData(width: 128, quality: 95)
       }
     }
+    sysinternalsLogo: file(relativePath: {regex: "/projects-logos/sysinternals.svg/"}) {
+        childImageSharp {
+          gatsbyImageData(width: 128, quality: 95)
+        }
+      }
```
Comment (vannyle) [inline src/pages/applications.js:94]:
```suggestion
   
```

**Review by vannyle — COMMENTED** [review]
Hi, could you please add the changes above? After that, the logo will appear on the page.

**Comment by MarioHewardt** [comment]
Suggested changes have been made.

**Comment by xmulligan** [comment]
Thank you for adding @MarioHewardt ! I hope you enjoy the holidays ☃️

**Pull request merged**